addons/website_sale_collect/tests/common.py

```diff
@@ -2,27 +2,38 @@
 
 from odoo.fields import Command
 
+from odoo.addons.delivery.tests.common import DeliveryCommon
 from odoo.addons.payment_custom.tests.common import PaymentCustomCommon
-from odoo.addons.website_sale.tests.common import WebsiteSaleCommon
+from odoo.addons.website_sale_stock.tests.test_website_sale_stock_product_warehouse import \
+    TestWebsiteSaleStockProductWarehouse
 
 
-class OnSiteCommon(PaymentCustomCommon, WebsiteSaleCommon):
+class InStoreCommon(PaymentCustomCommon, TestWebsiteSaleStockProductWarehouse, DeliveryCommon):
```

Comment:
Do not inherit a test class in a common, you should only inherit commons.
If you inherit `TestWebsiteSaleStockProductWarehouse`, the test methods in `TestWebsiteSaleStockProductWarehouse` will be run:
* for `TestWebsiteSaleStockProductWarehouse`
* for `InStoreCommon`
* for all classes inheriting `InStoreCommon`